docs/Operators.md

```diff
@@ -7079,7 +7079,7 @@ This version of the operator has been available since version 1 of the default O
 
 <dl>
 <dt><tt>input</tt> : T</dt>
-<dd>1-D input tensor</dd>
+<dd>Input tensor</dd>
```

Comment:
This is an auto-generated document. Please fix the description in the related defs.cc file and run /onnx/defs/gen_doc.py for updating this document. Thank you!

Reply:
ok - thanks. just did it!